evap/static/ts/tests/frontend/staff-evaluation-edit.ts

```diff
@@ -1,40 +0,0 @@
-import { test, expect } from "@jest/globals";
-import { ElementHandle } from "puppeteer";
-
-import { pageHandler } from "../utils/page";
-import { assertDefined } from "../../src/utils";
-
-// regression test for #1769
-test(
-    "changes form data",
-    pageHandler("/staff/semester/PK/evaluation/PK/edit/normal.html", async page => {
-        const managerId = await page.evaluate(() => {
-            const tomselect = (document.getElementById("id_contributions-0-contributor") as any).tomselect;
-            const options = tomselect.options;
-            const managerOption = Object.keys(options).find(
-                key => options[key].text == "manager (manager@institution.example.com)",
-            );
-            tomselect.setValue(managerOption);
```

Comment:
Is this selection important? I can't find it in the Python version

Reply:
Ah, I suppose this is due to https://github.com/e-valuation/EvaP/pull/2077#discussion_r1922778892 ?

Reply:
no, the comment is unrelated. I'm looking into it..

Reply:
I think this got replaced by a simple managerId update. Reading the referenced PR, I think that it may be important. I check if we can have something similar in python with selenium.

Reply:
done something similar now that should be equivalent :+1: